client/view/showStats.js

```diff
@@ -0,0 +1,141 @@
+    itemElem.setAttribute('style', `
+    --p: ${item.coverage};
+    --a: ${alpha};
+    `)
+    if(item.coverage > 10) {
+      itemElem.innerHTML = item.name;
+      itemElem.classList.add('BrowsersStat__globalCoverageBarItem--texted')
+    }
+    element.appendChild(itemElem);
+  })
+}
+
+const statsPlaceholder = document.querySelector('.BrowsersStat__placeholder');
+
+export function hideStatsPlaceholder() {
+  if(statsPlaceholder.classList.contains('BrowsersStat__placeholder--hidden')) {
+    return true;
+  }
+
+  const statsElem = document.querySelector('.BrowsersStat__stat');
+  statsElem.classList.remove('BrowsersStat__stat--hidden');
+
+  statsPlaceholder.classList.add('BrowsersStat__placeholder--hidden');
+
+  return true;
+}
+
+function _getWikipediaLink(id) {
+  return WIKIPEDIA_URL + wikipediaLinks[id]
+}
+
+function _createCoverageCell(coverage) {
+  const coveragePercentageHtmlString = (cov) => cov + '%';
+  const coveragePercentageCssString = (cov) => {
+    const result =  Math.log(1 + cov) * 100 / Math.log(1 + 100);
+    if (result === 0) {
+      return '0';
+    } else if (result > 5) {
+      return result + '%';
+    }
+    return '1px';
+  }
+
+  const coverageCell = document.createElement('td');
+  coverageCell.classList.add('BrowsersStat__td');
+  coverageCell.innerHTML = coveragePercentageHtmlString(coverage);
+  coverageCell.classList.add('BrowsersStat__td--coverage');
+  coverageCell.setAttribute('style', `--c:${coveragePercentageCssString(coverage)}`);
+  return coverageCell;
+}
+
+function _createVersionCell(version) {
+  const versionCell = document.createElement('td');
+  versionCell.classList.add('BrowsersStat__td');
+  versionCell.innerHTML = version;
+  return versionCell;
+}
+
+export function updateBrowsersStats(data) {
+  const element = document.getElementById('browsers-stats');
+
+  const table = document.createElement('table');
+  table.classList.add('BrowsersStat__table');
+
+  data.forEach(({id, name, versions: versionsInput}) => {
+
+    const versions = Object.entries(versionsInput)
+      .sort(([, coverageA], [, coverageB]) => coverageB - coverageA)
+      .map(([ version, coverage ]) => {
+        return {
+          version,
+          coverage,
+        }
+      });
+
+    const tBody = document.createElement('tbody');
+    const tr = document.createElement('tr');
+    tr.classList.add('.BrowsersStat_tr');
+
+    const iconCell = document.createElement('td');
+    iconCell.classList.add('BrowsersStat__td');
+    const iconElem = document.createElement('img');
+    iconElem.classList.add('BrowsersStat__icon');
+    iconElem.src = `/assets/browser_logos/${id}.svg`;
+    iconCell.setAttribute('rowspan', versions.length);
+    iconCell.appendChild(iconElem);
+    tr.appendChild(iconCell);
+
+    const nameCell = document.createElement('td');
+    nameCell.classList.add('BrowsersStat__td');
+    const nameLink = document.createElement('a');
+    nameLink.classList.add('BrowsersStat__link', 'Link');
```

Comment:
BEM in my projects has a policy to have only 1 block on a tag. What do you think of using it here as well?

I'm afraid that block composition destroy the idea of block isolation. You need to know `Link` classes to make `BrowsersStat__link`. So on any `Link` changes, you can break `BrowsersStat__link`.

For instance, we can replace it with modifier:

```js
nameLink.classList.add('Link', 'Link--simple');
```